python/pyarrow/tests/test_pandas.py

```diff
@@ -2272,6 +2272,30 @@ def test_to_pandas(self):
         series = pd.Series(arr.to_pandas())
         tm.assert_series_equal(series, expected)
 
+    def test_to_pandas_multiple_chunks(self):
+        # ARROW-11855
+        bytes_start = pa.total_allocated_bytes()
```

Comment:
Probably want to call `gc.collect()` just before this, to avoid false positives.

Reply:
Done.